specs/latest/2.0/index.html

```diff
@@ -2123,6 +2123,9 @@ <h2 class="no-toc">Table of contents</h2>
         </table>
         <p>If <em>pname</em> is not in the table above, generates an <code>INVALID_ENUM</code> error.</p>
         <p>If <em>sampler</em> is not a valid sampler object, generates an <code>INVALID_OPERATION</code> error.</p>
+        <p>Modifying a parameter of a sampler object affects all texture units to which that sampler object is bound.
+        Calling <code>TexParameter</code> has no effect on the sampler object bound to the active texture unit.
+        It will modify the parameters of the texture object bound to that unit.</p>
```

Comment:
I am not sure if this text adds value to the spec.  WebGL 2 spec is based on top of ES 3.0 spec, and this text is something fundamental about how sampler objects work, and it should be very clear in the ES 3.0 spec.  I see no reason to repeat that in the WebGL 2 spec.